setup.py

```diff
@@ -8,9 +8,6 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-if sys.version_info[0] < 3:
-    raise NotImplementedError("Do not support version 2 of python")
-
```

Comment:
`python_requires` already strictly enforces this, so this is essentially dead code so remove it.